### DISCUSSION TITLE
LivelyWallpaper is on top, so you can't see the desktop icons

### DISCUSSION THREAD

**Discussion by talkgrio** [rocksdanister/lively#2129] [General]
One day, LivelyWallpaper suddenly came to the top of my desktop and I couldn't see any desktop icons. The taskbar is also not displayed. The app prompts me to prioritize controls in the performance options, but I've tried them all. But Rainmater appears at the top.

**Comment by rocksdanister** [comment]
dup: https://github.com/rocksdanister/lively/issues/2074
